torch/_inductor/ir.py

```diff
@@ -4447,15 +4433,8 @@ def codegen_collective(self, wrapper, output_name, input_names):
             f"{output_name}, {input_names[0]}, async_op=True, group={output_name}_pg)"
         )
 
-        # At this point, output_name points to a fresh buffer
```

Comment:
is my diff  view screwed up? it looks like there was a duplicate call to dist.* under this codegen_collective and you're just cleaning it up.  (but also removing the register_work part)

Reply:
ok, switched to just reading the ir.py - makes a lot more sense now. nice design.